core/src/main/java/com/sequenceiq/cloudbreak/service/sharedservice/SharedServiceConfigProvider.java

```diff
@@ -137,6 +137,10 @@ private void addDatalakeRequiredProperties(Set<String> datalakeProperties) {
         datalakeProperties.add("ranger.audit.solr.zookeepers");
         datalakeProperties.add("atlas.rest.address");
         datalakeProperties.add("atlas.kafka.bootstrap.servers");
+        datalakeProperties.add("atlas.kafka.security.protocol");
```

Comment:
@schfeca75 do we still need this method?

Reply:
@akanto This method only used from ClusterV1Controller. I recommend to remove it after the demo during the cleanup task.

Reply:
ok